test/e2e/egressip.go

```diff
@@ -871,6 +873,323 @@ spec:
 		}
 	})
 
+	// Validate the egress IP when a pod is managed by more than one egressIP object
+	/* This test does the following:
+	   0. Add the "k8s.ovn.org/egress-assignable" label to node2 (pod2Node/egress1Node)
+	   1. Create one pod matching the EgressIP: running on node2 (pod2Node/egress1Node)
+	   2. Create an EgressIP object1 with two egress IP's - egressIP1 and egressIP2 defined
+	   3. Check that the status is of length one and that one of them is assigned to node2 (pod2Node/egress1Node) while other is pending
+	   4. Check connectivity from pod to an external container and verify that the srcIP is the expected egressIP from object1
+	   ----
+       5. Create an EgressIP object2 with one egressIP3 defined (standby egressIP)
+	   6. Check that the second egressIP object is assigned to node2 (pod2Node/egress1Node)
+	   7. Check the OVN DB to ensure no SNATs are added for the standby egressIP
+	   8. Check connectivity from pod to an external container and verify that the srcIP is the expected egressIP from object1
+	   ----
+	   9. Delete assigned egressIP1 from egressIP object1
+	   10. Check that the status is of length one and that standby egressIP3 of egressIP object2 is assigned to node2 (pod2Node/egress1Node)
+	   11. Check connectivity from pod to an external container and verify that the srcIP is the expected standby egressIP3 from object2
+	   12. Check the OVN DB to ensure SNATs are added for only the standby egressIP3
+	   ----
+	   13. Mark egress2Node as assignable and egress1Node as unassignable
```

Comment:
yeah will do, its actually confusing because we do:
```
egress1Node = node{
			name:   nodes.Items[1].Name,
			nodeIP: ips[1],
		}
		egress2Node = node{
			name:   nodes.Items[2].Name,
			nodeIP: ips[2],
		}
		pod1Node = node{
			name:   nodes.Items[0].Name,
			nodeIP: ips[0],
		}
		pod2Node = node{
			name:   nodes.Items[1].Name,
			nodeIP: ips[1],
		}
```